### PR TITLE
Add docs for environ_name functionality

### PR DESCRIPTION
**Pull request by lwm**
Closes https://github.com/jazzband/django-configurations/issues/173.

### CODE REVIEW

**Comment by lwm** [comment]
Bump! Any review on this one folks? Thanks :+1:

File: docs/values.rst [hunk 1]
```diff
@@ -120,6 +120,21 @@ instead of using the name of the :class:`~Value` instance.::
     class Dev(Configuration):
         TIME_ZONE = values.Value('UTC', environ_name='MYSITE_TZ')
 
+Allow final value to be used outside the configuration context
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
Comment (blueyed) [inline docs/values.rst]: Should be aligned with the heading, also below.

File: docs/values.rst [hunk 2]
```diff
@@ -120,6 +120,21 @@ instead of using the name of the :class:`~Value` instance.::
...
+
+You may use the ``environ_name`` parameter to allow a :class:`~Value` be
```
Comment (blueyed) [inline docs/values.rst]: s/be/to be/?

File: docs/values.rst (hunk 1 above)
Comment (lwm) [inline docs/values.rst]: Uhh, I'm not sure what you mean here?

File: docs/values.rst (hunk 2 above)
Comment (lwm) [inline docs/values.rst]: :+1:

File: docs/values.rst (hunk 1 above)
Comment (blueyed) [inline docs/values.rst]: Remove two `^` to have the same length as the text.

Comment (lwm) [inline docs/values.rst]: Wait, but ... the line of `^`s are the same length as the text!?

Comment (blueyed) [inline docs/values.rst]: Sorry, there seems to be a problem with the font config in Qutebrowser for me:
2017-07-16-160832_663x350_scrot

It looks fine in Firefox, so sorry for the confusion.

**Comment by blueyed** [comment]
Thanks!

File: docs/values.rst (hunk 1 above)
Comment (blueyed) [inline docs/values.rst]: It was caused by using different hintstyle (hintnone) for bold fonts (which the line of `^`s is for some reason via GH's css - but the line above is not).
Ref: https://github.com/blueyed/dotfiles/blob/7c4d7be86beea22dc7e230762de93790617e16f6/config/fontconfig/fonts.conf#L151

This should not really cause the width to become different, which might be a bug in fontconfig or the webengine backend in Qutebrowser.

**Pull request merged**